andela/travel_tool_front
# #160717378 Edit travel team member

#### What does this PR do?
- The Travel admin can change the center of an existing travel team member

#### Description of Task to be completed?
- Create actions, sagas and reducers for changing Andela centers
- The travel admin can  see the list of travel team members
- The travel admin can click the edit button to change the center of an existing travel team member

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Change into the project directory
- Run the server `make start`
- Make sure you are a Travel admin or super admin. Click on the edit button to change the center of an existing travel team member 


#### Any background context you want to provide?
- You have to be a travel admin or super admin to see the link and access the component 


#### What are the relevant pivotal tracker stories?
[160717378](https://www.pivotaltracker.com/story/show/160717378)

#### Screenshots (if appropriate)

![screenshot 2018-10-18 at 5 51 55 pm 2](https://user-images.githubusercontent.com/20597418/47170670-8df57500-d2fe-11e8-88e5-550cd4c2fc12.png)


#### Questions:
- None
